contrib/mesos/pkg/minion/server.go

```diff
@@ -204,6 +207,18 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args *[]string, log
 		log.Infof("wrote %d bytes to %v", written, logFileName)
 	}()
 
+	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
+	if ms.pathOverride != "" {
+		env := os.Environ()
+		cmd.Env = make([]string, 0, len(env))
+		for _, e := range env {
+			if strings.HasPrefix(e, "PATH=") {
```

Comment:
In fact it replaces it now. But you are right, it should just drop it the old and append it after the for loop.